C_CPP/Data Structures/Binary Tree/Binary_Tree_sum_of_nodes_at_kth_level.md

```diff
@@ -0,0 +1,121 @@
+# Sum of all nodes at Kth level in Binary Tree - <br/>
+Consider a binary tree as -  <br/>
```

Comment:
```suggestion
Consider a binary tree as -   
```

Avoid using HTML tags in md unless inevitable cz they tend to break
Convention for line break is 2 spaces at end of the line